scripts/generators/es_template.py

```diff
@@ -38,13 +38,14 @@
 def generate(
     ecs_nested: Dict[str, FieldNestedEntry],
     ecs_version: str,
+    ecs_component_name_prefix: str,
     out_dir: str,
     mapping_settings_file: str,
     template_settings_file: str
 ) -> None:
```

Comment:
Not really having context for this tool, adding a new argument like this in the middle seems like it might not be the most compatible way of doing this.. Usually I'd expect new arguments to show up in the `kwargs` section of the list and then have a reasonable default, which in this case looks like it should be `ecs`:

```suggestion
def generate(
    ecs_nested: Dict[str, FieldNestedEntry],
    ecs_version: str,
    out_dir: str,
    mapping_settings_file: str,
    template_settings_file: str,
    ecs_component_name_prefix: str = "ecs",
) -> None:
```

Reply:
Moved to kwarg

Reply:
LGTM